src/containers/layout/navbar/NavBar.jsx

```diff
@@ -60,10 +59,10 @@ const Navbar = () => {
   return (
     <Box sx={styles.header}>
       <Button
-        component={Link}
+        component={HashLink}
         size='small'
         sx={styles.logoButton}
-        to={guestRoutes.home.path}
+        to={guestRoutes.home.path + guestRoutes.welcome.path}
```

Comment:
u can do like routs are done in navbar so concatination will be no needed

Reply:
> u can do like routs are done in navbar so concatination will be no needed

left only one route path as done in the navbar